### PR TITLE
Update user guide to reflect v1.0 features

### PR DESCRIPTION
**Pull request by Emkay16**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,42 +1,150 @@
...
+   
+1. Open a terminal/command line (cmd)/powershell. A Windows 10 OS' screenshot is here:\
+   \
+   ![PowerShell](./images/WindowsPowerShell.png)
+   
+1. Execute `java -jar tp.jar` to start the Patient Manager.
 
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
+1. Once the welcome message appears, simply type in a command (e.g. [`help`](#print-a-help-message-help)) and hit `ENTER` at the end.
```
Comment (fsgmhoward) [inline docs/UserGuide.md:42]: Probably a renumbering is required (1. 2. 3. 4. ...)?

**Review by fsgmhoward — CHANGES_REQUESTED** [review]
Looks really good! Just that a minor point needs to be fixed.

File: docs/UserGuide.md (hunk 1 above)
Comment (Emkay16) [reply docs/UserGuide.md:42]: Initially I had them labelled 1. 2. 3. but according to the [markdown coding standard](https://se-education.org/guides/conventions/markdown.html), using generic numbers (1.) makes it easier to insert more items in the future.

Comment (fsgmhoward) [reply docs/UserGuide.md:42]: Oh. I notice that. I will merge it then. Thanks!

**Review by fsgmhoward — APPROVED** [review]

**Pull request merged**